src/adapter/bun/conninfo.test.ts

```diff
@@ -0,0 +1,57 @@
+import { Context } from '../../context'
+import { HonoRequest } from '../../request'
+import { getConnInfo } from './conninfo'
+
+const createRandomBunServer = () => {
+  const address = Math.random().toString()
+  const port = Math.floor(Math.random() * (65535 + 1))
+  return {
+    address,
+    port,
+    server: {
+      requestIP() {
+        return {
+          address,
+          family: 'IPv6',
+          port,
+        }
+      },
+    },
+  }
+}
+describe('getConnInfo', () => {
+  it('Should info is valid', () => {
+    const { port, server, address } = createRandomBunServer()
+    const c = new Context(new HonoRequest(new Request('http://localhost/')), { env: server })
+    const info = getConnInfo(c)
+
+    expect(info.remote.port).toBe(port)
+    expect(info.remote.address).toBe(address)
+    expect(info.remote.addressType).toBe('IPv6')
+    expect(info.remote.transport).toBeUndefined()
+  })
+  it('Should getConnInfo works when env is { server: server }', () => {
+    const { port, server, address } = createRandomBunServer()
+    const c = new Context(new HonoRequest(new Request('http://localhost/')), { env: { server } })
+
+    const info = getConnInfo(c)
+
+    expect(info.remote.port).toBe(port)
+    expect(info.remote.address).toBe(address)
+    expect(info.remote.addressType).toBe('IPv6')
+    expect(info.remote.transport).toBeUndefined()
+  })
+  it('Should throw error when user didn\'t give server', () => {
```

Comment:
A simple solution. How about re-writing `didn\'t` to `did not`?

Reply:
Thanks, I do it.
However, I think it's not a fundamental solution, it looks like coping therapy.

This discussion is out-topic, but I think we should open issue to resolve it.

Reply:
> This discussion is out-topic, but I think we should open issue to resolve it.

Yeah! Exactly.

Reply:
I'll mark this problem as resolved.